content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue.md

```diff
@@ -76,6 +76,12 @@ Once you have enabled the "Require merge queue" setting, you can also access the
 -
   **Merge limits**: Select the minimum and maximum number of pull requests to merge in a single group (between `1` and `100`), and a timeout after which the queue should stop waiting for more entries and merge with fewer than the minimum number of pull requests. Exactly how many PRs are included in a group depends on a merge queue's settings:
```

Comment:
Hi. I chatted with the engineers and we think that adjusting the description here will address this issue:

```suggestion
  **Merge limits**: Select the minimum and maximum number of pull requests to merge into the base branch at the same time (between `1` and `100`), and a timeout after which the queue should stop waiting for more entries and merge with fewer than the minimum number.
```

Reply:
Looks like I wasn't able to prevent this from merging, here's what I would propose to remove the added note.

```suggestion
  **Merge limits**: Select the minimum and maximum number of pull requests that have completed required checks to merge into the base branch at the same time (between `1` and `100`), and a timeout after which the queue should stop waiting for more entries and merge with fewer than the minimum number.
```

In the interest of brevity (and lack of personal experience with it) I am not getting into the interaction between these settings and **Only merge non-failing pull requests**, but this at least makes it explicit that merge groups after the required checks (which helps imply the point of the original note which is that PRs are not grouped for build).